etc/db_schema_main.sql

```diff
@@ -26,3 +26,6 @@ SOURCE etc/db_schema_user_role_table.sql;
 
 /* #4: Create User session table */
```

Comment:
I'm dumb, these numbers were a bad idea. We are taking them out of the master branch. You can kill them XD